packages/host/app/services/realm.ts

```diff
@@ -371,6 +379,25 @@ export default class RealmService extends Service {
     return resource.token;
   }
 
+  async fetchPublicRealmURLs(): Promise<string[]> {
+    let realmServerURL = new URL(window.location.href);
+    if (ENV.environment === 'development' || ENV.environment === 'test') {
+      realmServerURL = new URL('http://localhost:4201');
```

Comment:
This kind of conditional is starting to appear in different parts of the code base. We need one place to deal with it (perhaps the realm-server service?)

Reply:
it is actually a property available in the new realm-server service in my PR